src/popups/InfoFPWPopup/InfoFPWPopup.js

```diff
@@ -0,0 +1,114 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Feature from 'ol/Feature';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { compose } from 'lodash/fp';
+import Button from 'react-spatial/components/Button';
+import { IoIosArrowRoundBack, IoIosArrowRoundForward } from 'react-icons/io';
+
+import './InfoFPWPopup.scss';
+
+const propTypes = {
+  feature: PropTypes.instanceOf(Feature).isRequired,
+  language: PropTypes.string.isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+class InfoFPWPopup extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imageIndex: 0,
+    };
+  }
+
+  setImageIndex(index) {
+    this.setState({
+      imageIndex: index,
+    });
+  }
+
+  render() {
+    const { feature, language, t } = this.props;
+    const { imageIndex } = this.state;
+
+    let text;
+    let image;
+    let pagination;
+
+    const pureText = feature.get(`${language}_text`) || feature.get('de_text');
+    if (pureText) {
+      // eslint-disable-next-line react/no-danger
+      text = <div dangerouslySetInnerHTML={{ __html: pureText }} />;
+    }
+
+    const photoPath = feature.get('photo_path');
+    if (photoPath) {
+      const photoPaths = [JSON.parse(photoPath), JSON.parse(photoPath)].flat();
+
+      image = (
+        <img
+          src={`${process.env.REACT_APP_CARTARO_1_URL}/${photoPaths[imageIndex]}`}
+          draggable="false"
+          alt={t('Kein Bildtext')}
+        />
+      );
+
+      if (photoPaths.length > 1) {
+        pagination = (
+          <div className="wkp-pagination-wrapper">
```

Comment:
Currently, I'm using the same classes from the component `<FeatureInformation>`. So, we do not have to change styling at two places. However, I don't think that this is good code style. What about a pagination component?

Reply:
> I have integrated the react-slideshow-image lib.

So, this is obsolete.